composer.lock

```diff
@@ -1,4213 +0,0 @@
-{
```

Comment:
keep this file in the git for Dependabot

Reply:
done